db/db.js

```diff
@@ -11,12 +11,37 @@ var database = mongoose.connection;
 var orderSchema = new Schema({
   customerName:{
     type: String,
+    var testName=validator.isAlphanumeric(str [, locale]);
+    if (testname===false){
+      message:'{Value} is not a valid name, please enter again'
+    }
+    else{
     required:[true, 'Please enter a customer name']
+  }
   },
+
   customerEmail: {
     type: String,
-    required:[true, 'Please enter a customer email']
-  customerPhone: String,
+    var testEmail=validator.isEmail(v);
+    if (testEmail===false){
+      message:'{Value} is not a valid email, please enter again'
+    }
+    else {
+      required:[true, 'Please enter a customer email']
+    }
+}
+  customerPhone:{
+    type: String,
+    Validate:{
+    validator: function(v){
```

Comment:
see here how the syntax is validator: function(v){} ? this is how the other custom validators need to be so like

validator: function(v){
// code that tests stuff on v
}